pkgs/applications/window-managers/dwm/dwm-status.nix

```diff
@@ -15,11 +20,17 @@ rustPlatform.buildRustPackage rec {
     dbus
     gdk_pixbuf
     libnotify
+    makeWrapper
     pkgconfig
   ];
 
   cargoSha256 = "0169k91pb7ipvi0m71cmkppp1klgp5ghampa7x0fxkyrvrf0dvqg";
 
+  postInstall = ''
+    wrapProgram $out/bin/${name} \
```

Comment:
Is `${name}` really the right thing to do here?

Is the binary called `dwm-status-0.4.0` or is it called `dwm-status`?